auction-server/src/opportunity/api.rs

```diff
@@ -219,7 +238,7 @@ pub async fn post_quote(
     State(store): State<Arc<StoreNew>>,
     Json(params): Json<QuoteCreate>,
 ) -> Result<Json<Quote>, RestError> {
-    if params.get_user_wallet_address() == Some(INDICATIVE_PRICE_TAKER) {
+    if params.get_user_wallet_address() == Some(generate_indicative_price_taker()) {
```

Comment:
This check doesn't make sense anymore. Instead, we should check the prefix of the user wallet address, the first 24 bytes.